index.html

```diff
@@ -92,6 +92,11 @@ <h3>
             <i class="fa fa-mouse-pointer"></i>
             Click for Demo
         </button>
+        <br>
```

Comment:
Let's wrap this `<br>` and `<button>` inside a `<div>` with an `id`, then in the JS we can use that div's id instead of `#focus` when hiding & showing the button. This way we can also hide the extra whitespace